src/Microsoft.Data/StringColumn.cs

```diff
@@ -255,5 +256,30 @@ private StringColumn Clone(PrimitiveColumn<long> mapIndices = null, bool invertM
             }
             return ret;
         }
+
+        public override Dictionary<TKey, ICollection<long>> HashColumnValues<TKey>()
```

Comment:
`HashColumnValues` => what does "hash" have to do with this method? My understanding of the method is that it builds a map of values => rowIndices.  It doesn't really "hash" the values.

Reply:
Oh, I guess I associate dictionary with hashing. I changed it to "GroupColumnValues" now. Open to other names too.